src/services/review/ReviewRequestService.ts

```diff
@@ -434,6 +434,11 @@ export default class ReviewRequestService {
     await reviewRequest.save()
   }
 
+  deleteReviewRequestApproval = async (reviewRequest: ReviewRequest) => {
+    reviewRequest.reviewStatus = "OPEN"
```

Comment:
```suggestion
    reviewRequest.reviewStatus = ReviewRequestStatus.Open
```

Minor nit.

Reply:
good catch! think it got through cos of rebase @_@